.nodes.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+HN=$(hostname)
+
+if [ "$HN" == "ccf-ci-5" ]; then
+    echo "-n 10.0.0.6 -n 10.0.0.8 -n 10.0.0.11"
```

Comment:
Can we use hostnames here? ie `-n ccf-ci-6 -n ccf-ci-7 -n ccf-ci-8`?

Reply:
No, they resolve to public IPs.